domains/abhinav.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "abhinavshyju"
+    },
+    "record": {
+        "CNAME": "portfolio-6ns.pages.dev"
```

Comment:
this seems like a default page for a astro app template

Reply:
<html>
<body>
<!--StartFragment-->
Error: Failed to publish your function. Got error: Uncaught ReferenceError: MessageChannel is not defined   at chunks/_@astro-renderers_4GnOaUIv.mjs:6534:16 in requireReactDomServer_browser_production   at chunks/_@astro-renderers_4GnOaUIv.mjs:12531:8 in requireServer_browser   at chunks/_@astro-renderers_4GnOaUIv.mjs:12543:29
--


<!--EndFragment-->
</body>
</html> Im stuck on this error. i will fix it asap